pennylane/qchem/molecule.py

```diff
@@ -72,8 +72,16 @@ def __init__(
         normalize=True,
     ):
 
-        if basis_name not in ["sto-3g", "STO-3G", "6-31g", "6-31G"]:
-            raise ValueError("Currently, the only supported basis sets are 'sto-3g' and '6-31g'.")
+        if basis_name.lower() not in [
+            "sto-3g",
+            "6-31g",
+            "6-311g",
+            "cc-pvdz",
+        ]:
+            raise ValueError(
+                "Currently, the only supported basis sets are 'sto-3g', '6-31g', '6-311g' and"
```

Comment:
Please also add the new basis set names here in the class docstring. 
<img width="676" alt="image" src="https://user-images.githubusercontent.com/23234279/201963097-718e28ec-0931-4d89-8e88-55b01aaba770.png">

Reply:
Added.